packages/rocketchat-graphql/server/api.js

```diff
@@ -0,0 +1,77 @@
+import { graphqlExpress, graphiqlExpress } from 'apollo-server-express';
+import { JSAccountsContext as jsAccountsContext } from '@accounts/graphql-api';
+import { SubscriptionServer } from 'subscriptions-transport-ws';
+import { execute, subscribe } from 'graphql';
+import { Meteor } from 'meteor/meteor';
+import { WebApp } from 'meteor/webapp';
+import bodyParser from 'body-parser';
+import express from 'express';
+import cors from 'cors';
+
+import { executableSchema } from './schema';
+
+const subscriptionPort = RocketChat.settings.get('Graphql_Subscription_Port') || 3100;
+
+// the Meteor GraphQL server is an Express server
+const graphQLServer = express();
+
+if (RocketChat.settings.get('Graphql_CORS')) {
```

Comment:
What happens if this setting is changed after the server has started? Does it require a restart of the server to have an effect?

Reply:
No, we added a middleware to check if it is enabled, this allow us to change the settings without a restart.